src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ProductRepository.php

```diff
@@ -452,56 +453,37 @@ public function hasAttributeInVariantGroup($productId, $attributeCode)
      */
     public function findProductIdsForVariantGroup(GroupInterface $variantGroup, array $criteria = [])
     {
-        $qb = $this->findAllForVariantGroupQB($variantGroup, $criteria);
-        $qb->select('Product.id');
+        $queryBuilder = $this->findAllForVariantGroupQB($variantGroup, $criteria);
 
-        return $qb->getQuery()->getResult();
+        return $queryBuilder->execute();
```

Comment:
now you return a curosr and not IDs
but let's say I didn't see nothing as it's about variant groups